packages/helpers/src/media.ts

```diff
@@ -0,0 +1,63 @@
+import { BreakpointType, breakpoints } from '../../tokens/src/global/breakpoints';
+
+import { css } from 'styled-components';
+
+import hide from './hide';
+import { Matcher, Media, Width } from './types';
+
+const availableBreakpoints = Object.keys(breakpoints) as BreakpointType[];
```

Comment:
💅 This feels redundant hence you defined a const `BREAKPOINTS_KEYS` with the same value.
```suggestion
const availableBreakpoints = BREAKPOINTS_KEYS;
```